gymnasium/vector/vector_env.py

```diff
@@ -430,6 +446,19 @@ def render_mode(self) -> tuple[RenderFrame, ...] | None:
         """Returns the `render_mode` from the base environment."""
         return self.env.render_mode
 
+    @property
+    def np_random(self) -> np.random.Generator:
```

Comment:
Do we need/want this? I don't remember what the general philosophy was regarding wrappers having their own np_random, but quietly (and unavoidably) passing on the unwrapped env's generator seems rather risky and unnecessarily opaque to me

Reply:
This is the current approach for `Env`, such that wrappers technically have no `np_random`, it is always the unwrapped envs.
Otherwise, you can get the same confusion when setting a `np_random` as this doesn't update the unwrapped environnments
This has been partially fixed with removing `__getattr__` but this could still cause confusion I think

Reply:
Name shadowing in the wrappers is among the most confusing things I've ever seen, I remember spending hours searching for bugs due to this in early gym versions. I'm happy it's gone, the wrappers should never have name-shadowing attributes